kernel/libc/koslib/inet_ntoa.c

```diff
@@ -20,7 +20,7 @@ char *inet_ntoa(struct in_addr addr) {
            values. This works, regardless of the endianness of the host system
            because the specs require the address passed in here to be in
            network byte order (big endian). */
-        part = ((uint8 *) &addr.s_addr)[i];
+        part = ((uint8_t *) &addr.s_addr)[i];
```

Comment:
These are very specifically because the 'uint8' is only defined in the arch/types, whereas _t are in the general types. Either way we are trying to move towards _t everywhere, to my understanding.